Cargo.toml

```
@@ -57,6 +57,7 @@ wasm = ["wasm-bindgen"]
 dump_lookahead_data = ["byteorder", "image"]
 
 [dependencies]
+histogram = "*"
```

Would be safer to have explicit version :)
```suggestion
histogram = "0.6.9"
```